conf/config_template.yaml

```diff
@@ -8,11 +8,13 @@
 #   7) Visualization parameters
 
 # Global parameters
-
 global:
   samples_size: 512
   num_classes: 5
   data_path: path/to/data
+  number_of_bands: 4
+  # Model available: unet, unetsmall, checkpointed_unet, ternausnet, fcn_resnet101, deeplabv3_resnet50, deeplabv3_resnet101
+  model_name: unet
```

Comment:
number of bands and model name seems to appear twice in this block. Line 14-17 and 19-20.